main.js

```diff
@@ -62,7 +62,7 @@ function setMenu(data) {
         menu.append(new MenuItem({ label: 'Today\'s Matches', enabled: false }));
 
         _.forEach(data, (match) => {
-            menu.append(new MenuItem({ label: getMatchTitle(match), click() {
+            menu.append(new MenuItem({ label: getMatchTitleWithScore(match), click() {
```

Comment:
We would need to do some kind of check here, as `match.time` is null if the game hasn't started (which is why I use `match.datetime` for upcoming matches). Really we want to show scores if the `match.status != 'future'`, otherwise use `getMatchTitle()`.